### DISCUSSION TITLE
display questions

### DISCUSSION THREAD

**Discussion by VintageEngineer** [egzumer/uv-k5-firmware-custom#403] [Q&A]
![PXL_20240109_214512418~2](https://github.com/amnemonic/Quansheng_UV-K5_Firmware/assets/38410268/dc12fa6f-a9d4-47a2-9b62-5ca52f186d80)
What are the circled indicators on the display?

```
R - is this redundant with the bold RX or does it mean something else?
><  - ?
I  - ?
```
addendum:
Here is the stock screen description:
<img width="572" alt="Screenshot 2024-01-09 at 6 58 40 PM" src="https://github.com/amnemonic/Quansheng_UV-K5_Firmware/assets/38410268/b3cbb3c4-dabc-43bf-86f1-873cc88da478">
"R" sometimes means "Reverse" but my radio wasn't set in Reverse mode. The "><" and "I"  are mysterious to me as well.

Might be nice to have a little guide for all the display icons for us newbies?

**Comment by dadanielat** [comment]
R seems to be redundant, but RX could be displayed up on A or down on B channel. '1 ><' or '2 ><' or '* ><' is displayed if scan mode/scan list is active.

**Comment by VintageEngineer** [comment]
> R seems to be redundant, but RX could be displayed up on A or down on B channel. '1 ><' or '2 ><' or '* ><' is displayed if scan mode/scan list is active.

R is confusing as it often means "Reverse". I don't understand your comments on the scan list... as the "><" symbol is at the top, not associated with one channel or the other? And the "I"?

**Comment by VintageEngineer** [comment]
Ok, I think the "R" and "T" is a General and redundant "Transmit" and "Receive" indicator.
I think these could be deleted and the space used for other stuff, given the giant "Rx" and "Tx" that also exists and is dominant.
The "><" and "I" are still mysterious to me...

**Comment by ErikS-web** [comment]
Don't use the QS manual.

We've made an [Online Manual in the Wiki](https://github.com/egzumer/uv-k5-firmware-custom/wiki/Radio-operation#memory-channels-scanning)

**Comment by VintageEngineer** [comment]
> Don't use the QS manual.
> 
> We've made an [Online Manual in the Wiki](https://github.com/egzumer/uv-k5-firmware-custom/wiki/Radio-operation#memory-channels-scanning)
Ok, so the "I" is for scan group "1"? 
Thank you!
Suggestion... Maybe consistently use Roman or Arabic numbers?
Any ideas on the "><"?

**Comment by ErikS-web** [comment]
👍 

The >< is always 4 seconds and what is after that on display. Probably DW. So DW will continue after the waittime, where it stays only on this VFO.

**Comment by jan-Sanku** [comment]
\>< does indeed seem to be DW related. It only shows up if some form of DW is on (main tx dual rx, and dual rx respond). It stays on for a few seconds after any transmission is received, and during that time only 1 channel will be watched. Once it's gone, both channels can rx again.

It also shows up if you're in FM radio mode (still only if dual watch is on), but you can still rx, so I'm not sure if that's intentional.

**Comment by egzumer** [comment]
The >< means dual watch is disabled

**Reply by VintageEngineer** [reply]
Not sure...
My testing shows in that area of the display:
**dual rx respond:** shows "DWR" and then shows "><" for 4 seconds after a transmit or receive
**main only:** shows nothing
**main tx dual rx:** shows "DW" and then "><" for 4 seconds after a transmit or receive
**crossband:** shows "XB" all the time

So to me it looks like the "><" indicates the time that you can respond on the same channel? 
But then it shouldn't show with "MainTX Dual RX" at all,, right?
Also, why show it after a transmit at all?

**Reply by egzumer** [reply]
The >< means dual watch is disabled, nothing less nothing more

**Reply by VintageEngineer** [reply]
That doesn't seem to be how it currently works:
dual rx respond: shows "DWR" and then shows "><" for 4 seconds after a transmit or receive
main only: shows nothing
main tx dual rx: shows "DW" and then "><" for 4 seconds after a transmit or receive
crossband: shows "XB" all the time

**Reply by egzumer** [reply]
Is this really so hard to understand? When it shows >< the DW function IS DISABLED, when it doesn't show it, it isn't. Crossband and main only doesn't use DW function.

**Reply by VintageEngineer** [reply]
I'm sorry , I really don't mean to be annoying! My apologies.

Maybe we have different definitions of "Dual Watch"? 
To me: "Dual Watch" is whenever both RECEIVERS are operating simultaneously, It does NOT have anything to do with transmitting. 
**"Dual Watch" DISABLED means only using one receiver** = main only. or crossband. 

Watching how the firmware works, it appears "><" is shown simultaneously when the ">" is shown on one of the channels. Is "><" shown any other time?

On the stock software ">" means: "The temporary transmitting channel, when the subchannel receives a call, it becomes the temporary transmitting channel. It appears "><" means the same thing, except it applies to both channels?

**Comment by VintageEngineer** [comment]
OK, I think I understand these all now, and I have submitted a request for changes that I think would make things a bit better.
Thank you!

**Comment by joc2** [comment]
Maybe a full description of all data that can be show on the screen, with picture, it will be easier for user to understand what does what or mean what .....
it can be a reference for user of the radio in field.... later

Something like this... that explain what is this and what can be show at this emplacement... picture is just to show what i talk about on 2 items
![image](https://github.com/egzumer/uv-k5-firmware-custom/assets/56229329/9b652331-02aa-4c65-a170-cd7a18144f61)

It's might be easier for all to see a picture and what is what...

 you can see from ijv how he explain his display, 
https://www.universirius.com/en_gb/preppers/quansheng-uv-k5-manuale-del-firmware-ijv/#Display

This is a sugestion to improve the product. :)
